sdk/src/main/java/co/omise/android/models/Capability.kt

```diff
@@ -0,0 +1,62 @@
+package co.omise.android.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import co.omise.android.api.Endpoint
+import co.omise.android.api.RequestBuilder
+import okhttp3.HttpUrl
+import org.json.JSONObject
+
+/**
+ * Represents Capabilities object and contains its {@link RequestBuilder}.
+ *
+ * @see <a href="https://www.omise.co/capability-api">Capabilities API</a>
+ */
+class Capability : Model {
```

Comment:
Can a `data class` extend from a normal java class? And do we gain anything here from using a data class?

Reply:
Yeah, it can but I think you might need refactor `Model` before change. So, the gain as you know ability of **data class** and I think it more suitable at here.

Reply:
BTW, we can keep it like this for now and do it in another PR.

Reply:
Okk if that works, it should be fine. Maybe I'll convert all model classes to Kotlin while changing Serialization and also make them data classes. 👍